plugins/kaniko.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kaniko
+spec:
+  version: "v1.1.0"
+  homepage: https://github.com/kvaps/kubectl-kaniko
+  shortDescription: Kaniko wrapper for kubectl
+  description: |
+    Build your Dockerfiles directly in your Kubernetes cluster.
+
+    Usage:
+
+      Show all kaniko commands
+      $ kubectl kaniko --help
+      
+      Build from current directory
+      $ kubectl kaniko --context . --no-push
+      
+      Specify namespace and kubeconfig
+      $ kubectl kaniko --context . --no-push --namespace default --kubeconfig ~/.kube/someconfig
+      
+      Login to remote registry
+      $ docker login docker.io
+      
+      Sort form
+      $ kubectl kaniko -c . -d docker.io/some/image:latest
+      
+      Use cache building
+      $ kubectl kaniko --context . --destination docker.io/some/image:latest --cache-repo docker.io/some/cache
+
+      Save image name and digest to file
+      $ kubectl kaniko --context . --destination docker.io/some/image:latest --digest-file /tmp/digest --image-name-with-digest-file /tmp/image
+      
+      Build from stdin
+      $ tar -cvf- . | kubectl kaniko --destination docker.io/some/image:latest --context tar://stdin
```

Comment:
The description is not the ideal place for your user docs. These examples are better suited for `kubectl kaniko --help`.

Reply:
Yeah, amount of examples reduced.